[2022-04-11 00:00:01,199] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C scheduled__2022-04-10T00:00:00+00:00 [queued]>
[2022-04-11 00:00:01,209] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C scheduled__2022-04-10T00:00:00+00:00 [queued]>
[2022-04-11 00:00:01,210] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 00:00:01,211] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-11 00:00:01,212] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 00:00:01,226] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): training_model_C> on 2022-04-10 00:00:00+00:00
[2022-04-11 00:00:01,231] {standard_task_runner.py:52} INFO - Started process 13403 to run task
[2022-04-11 00:00:01,235] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_C', 'scheduled__2022-04-10T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpn6ropte6', '--error-file', '/tmp/tmpshd8fvq3']
[2022-04-11 00:00:01,236] {standard_task_runner.py:80} INFO - Job 14: Subtask training_model_C
[2022-04-11 00:00:01,304] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.training_model_C scheduled__2022-04-10T00:00:00+00:00 [running]> on host c0c6f6bf1c08
[2022-04-11 00:00:01,364] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_C
AIRFLOW_CTX_EXECUTION_DATE=2022-04-10T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-10T00:00:00+00:00
[2022-04-11 00:00:01,366] {python.py:175} INFO - Done. Returned value was: 8
[2022-04-11 00:00:01,391] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_C, execution_date=20220410T000000, start_date=20220411T000001, end_date=20220411T000001
[2022-04-11 00:00:01,451] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-11 00:00:01,512] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
